src/pages/DelegatoryValidator/StakeDialog.tsx

```diff
@@ -49,6 +66,28 @@ export default function StakeDialog({
     setStakedAmount(amt);
   };
 
+  const handleSignAndSubmitTransaction = async () => {
+    const payload: Types.TransactionPayload = {
+      type: "entry_function_payload",
+      function: `${DELEGATION_POOL_ADDRESS}::delegation_pool::add_stake`,
+      type_arguments: [],
+      arguments: [validator.owner_address, Number(stakedAmount) * OCTA], // staked amount arg is in octa
+    };
+
+    try {
+      const response = await signAndSubmitTransaction(payload);
+      setTransactionInProcess(true);
+      await state.aptos_client.waitForTransaction(response?.hash!);
+      setTransactionHash(response?.hash!);
+      setTransactionSucceeded(true);
+      handleDialogClose();
+    } catch (error: any) {
+      // TODO(jill): figure out how to gracefully handle txn errors
+    } finally {
+      setTransactionInProcess(false);
+    }
+  };
```

Comment:
check this out https://github.com/aptos-labs/governance/blob/main/src/api/hooks/useSubmitTransaction.ts, let's handle the transaction following the governance codebase. feel free to leave a BE TODO here if you don't want to address it in this PR.